docs/faq.rst

```diff
@@ -109,7 +109,7 @@ following settings::
 Deleting a stale or broken build environment
 --------------------------------------------
 
-If you're having trouble getting your version to build, try wiping out the existing build/environment files.  On your version list page ``/projects/[project]/versions`` there is a "Wipe" button that will remove all of the files associated with your documentation build, but not the documentation itself.
+If you're having trouble getting your version to build, try wiping out the existing build/environment files.  On each version detail page ``/projects/[project]/version/[slug]`` there is a "wipe" link at the bottom of the page (next to the "Save" button) that will remove all of the files associated with your documentation build, but not the documentation itself.
```

Comment:
I'd just delete this and point it at the guide? Probably a useful thing to do with other FAQ's that we should write a guide about.

Reply:
I like the idea. Besides, there is no need to matain in two places the same instructions.

What are you thinking? Something just like 

    See: :doc:`guides/wipe-environment`

or something more elaborated?

Reply:
wfm

Reply:
Done.